app/views/users/index.html.erb

```diff
@@ -1,4 +1,15 @@
-<div class="container">
-    <h1 class="fs-1 fw-bold mb-4">Users</h1>
-    <%= render @users %>
-</div>
+<% @users.each do |user| %>
+    <div class='user-data'>
+        <div class="image-box">
+            <%= image_tag(user.photo, class: "user-image") %>
+        </div>
+        <div class='details-box'>
+            <p class='user-name'>
+                <%= link_to user.name, user_path(user) %>
+            </p>
+            <p class='no-of-posts'>
+                Number of posts: <%= user.posts_counter %>
```

Comment:
- [ ] Great job  enabled `:confirmable` is in the user model 👏🏻 However,I'm getting this messsage error page after a successful sign_inl. please solve this problem. 👍🏻 

![image](https://github.com/Ade179/Myblog/assets/79058364/cd63329f-c6fc-4756-994b-a88ce78a6d96)